.env.production.sample

```diff
@@ -134,3 +134,6 @@ STREAMING_CLUSTER_NUM=1
 # If you use Docker, you may want to assign UID/GID manually.
 # UID=1000
 # GID=1000
+
+# Maximum allowed character count
+# MAX_TOOT_CHARS=500
```

Comment:
afaik, toots are called statuses in code…

Reply:
You think it's worth to rename it again?

Reply:
I'm personally fine either ways. `MAX_STATUS_CHARS` would be more consistent with the source code, but the configuration option is intended for admins, not for developers.